hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/footer/DeleteActionsLayout.java

```diff
@@ -257,47 +256,42 @@ private void deleteTargetTag(final Component source) {
         }
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Same moving the unchecked `@SuppressWarnings("unchecked")` to line `261` the only unchecked operation in the method.